moon-store-mac-os/moon-store-mac-os/Services/BaseNetworkService.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  BaseNetworkService.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/3/24.
+//
+
+import Foundation
+
+// TODO: - Error handling
+
+private enum Constants {
+    static let scheme: String = "https"
+    static let baseUrl: String = "moon-store-production.up.railway.app"
+}
+
+actor BaseNetworkService {
```

Comment:
Alguna razón especifica para usar un actor?

Reply:
Oh thanks!

I forgot change it instead of `class` here @santelizvargas